src/pages/AnnualReport/AnnualReportForm/AnnualReportForm.tsx

```diff
@@ -47,8 +47,6 @@ const AnnualReportForm = (props: Props) => {
             e.keyCode !== 38 && // arrowUp
             e.keyCode !== 39 && // arrowRight
             e.keyCode !== 40 && // arrowDown
-            e.keyCode !== 188 && // comma
-            e.keyCode !== 190 && // period
```

Comment:
This approach is, at my opinion, the worst you could choose. Why do you hard code these values??? There are regexes invented for such purposes. Please, read about regexes and rewrite it to the normal code

Reply:
Ok, i will change it